dig_test.go

```diff
@@ -2468,24 +2468,56 @@ func assertCtorsEqual(t *testing.T, expected []*dot.Ctor, ctors []*dot.Ctor) {
 	}
 }
 
+func testParam(t reflect.Type, n string, g string, o bool) *dot.Param {
```

Comment:
optional: since these are helpers for testing in a test file, they don't need the `test` prefix. You can use a shorter name if you prefer.

Also, it looks like these are only used in TestDotGraph. You can declare anonymous functions and assign them to variables inside TestDotGraph if you're concerned about overly generic names polluting the namespace.

```
tparam := func(....) *dot.Param {
  ...
}

tresult := func(...) *dot.Result {
  ...
}
```